### ISSUE TITLE
Compute session is not starting in each lab that requires it

### ISSUE THREAD

**Issue by usamabinrauf** [MicrosoftLearning/mslearn-ai-studio#18]
### Description of issue:

Compute session is not starting and throwing an error after waiting for more than 30mins. This issue persists in each lab that require compute sessions.

![Screenshot_6](https://github.com/user-attachments/assets/c94e37ce-ed5f-4361-a904-b4e43b631e3a)

![Screenshot_7](https://github.com/user-attachments/assets/c153d950-fe7f-47d2-886f-9fd63bff2d11)

**Comment by usamabinrauf** [comment]
Issue resolved!

**Comment by usamabinrauf** [comment]
The issue came again.

**Comment by usamabinrauf** [comment]
Ignore this issue.

**Comment by usamabinrauf** [comment]
Facing this issue from the past week.

**Comment by afelix-95** [comment]
@ivorb I tested labs 03, 03b and 04 and while I didn't find the same error, it took way longer than what's expected for the compute sessions to start running. Although we get a notification in the studio saying that it takes 1-3 min for the session to start, for me it was taking 10-15 min for each lab. It seems like an intermittent issue that can be brought up to the product team.

**Comment by ivorb** [comment]
Agreed, this does seem like an intermittent product issue. I'll reach out to the product owners and will close this issue as it's not an error with the labs.

**Comment by usamabinrauf** [comment]
@ivorb I encountered a persistent issue when creating a project in the East US 2 region. However, when I switched to the East US region, the compute session started successfully on the first attempt.
